### DISCUSSION TITLE
[FEATURE REQUEST] Ability to have turbo on direction inputs

### DISCUSSION THREAD

**Discussion by akisinn** [OpenStickCommunity/GP2040-CE#569] [General]
Hi.
I wanna assign turbo button for lever input(down direction).
But on turbo setting of "Add-Ons", it seems it could assign for button input only. 
Can I do?

**Comment by bsstephan** [comment]
This sounds like a feature request, please make one in the GP2040-CE issues.

**Comment by TheTrainGoes** [comment]
Hi @akisinn ,

What is the use care for turbo on directions?
